lib/sales_engine.rb

```diff
@@ -15,4 +15,20 @@ def self.from_csv(data)
     SalesEngine.new(data[:items], data[:merchants], data[:invoices])
   end
 
+  def merchants
```

Comment:
I think we need to use the version I pushed, but I want to figure out how to merge that without overwriting the other work you've done... I think i can pull, review changes, decide which to keep and then re-push mine... that's probably what I will do.